internal/pkg/auth/storage.go

```diff
@@ -214,3 +226,28 @@ func createEncodedTextFile(activeProfile string) error {
 
 	return nil
 }
+
+// GetProfileEmail returns the email of the user associated with the given profile.
+// If the profile is not authenticated, it returns an empty string.
+func GetProfileEmail(profile string) string {
+	authFlow, err := getAuthFlowWithProfile(profile)
```

Comment:
I think it would be easier to read of this was inlined instead of calling a small new function that is just used here

Reply:
But it's also fine this way

Reply:
I agree, the original intention was to abstract the fetching of the authflow, but I don't think it will change and it is also done in this package